packages/backend/src/nest/libp2p/libp2p.service.ts

```diff
@@ -375,7 +383,7 @@ export class Libp2pService extends EventEmitter {
     if (params.instanceName != null) {
       this.logger = this.logger.extend(params.instanceName)
     }
-    this.logger.info(`Creating new libp2p instance`)
+    this.logger.info(`Creating new libp2p instance with psk`, params.psk)
```

Comment:
We should avoid logging secrets, especially things that give access like PSK, so that when users send us logs or when we post logs to issues we aren't making communities public inadvertently.